client/my-sites/media-library/filter-bar.jsx

```diff
@@ -148,6 +152,11 @@ export class MediaLibraryFilterBar extends Component {
 	}
 
 	renderPlanStorage() {
+		//hide the plan storage when viewing external sources
+		if ( this.props.source ) {
```

Comment:
Or, you could write `{ ! this.props.source && this.renderPlanStorage() }` in the `render()` method.